src/GeneralStateTestsFiller/stCreateTest/CREATE_ContractRETURNBigOffsetFiller.json

```diff
@@ -0,0 +1,44 @@
+{
+    "CREATE_ContractRETURNBigOffset" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "89128960",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "100000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" : {
+            "data" : "{ (RETURN 0x074ac2 0x0d15bc) }",
```

Comment:
multiple json comment fields should be prefixed with //

Reply:
@winsvega do you want `//`s before we merge this PR?

Reply:
well actually this should breake json scheme check in the test repo. but it is green.
@jwasinger ? 

the test itself is ok.

Reply:
thanks for the feedback @winsvega @pirapira , I'll add `//` before it is merged.

Reply:
@pirapira @winsvega I added the prefix `//`to the comment fields